src/neo/SmartContract/Callback.cs

```diff
@@ -0,0 +1,18 @@
+using Neo.VM;
+using Neo.VM.Types;
+using System;
+
+namespace Neo.SmartContract
+{
+    internal class Callback
```

Comment:
This should be an abstract class. We need two subclass: `PointerCallback` and `SyscallCallback`.

Reply:
We have problem with protected methods, `onSyscall` and `LoadContext` please take a look to the last commit

Reply:
https://github.com/neo-project/neo/pull/1629/commits/f1c749131b81a1ae441d9a638ac71e2f14a8ace1#diff-f3d5b66809ac28d481c2ea5032875043R33

and 

https://github.com/neo-project/neo/pull/1629/commits/f1c749131b81a1ae441d9a638ac71e2f14a8ace1#diff-81bcdc8edbf9734b2fd78f394eabd7eaR25